### PR TITLE
Fix namecheap lock status

### PR DESCRIPTION
**Pull request by nbebout**
This fixes the NameCheap lock status by calling namecheap.domains.getRegistrarLock for each domain.  @lfkeitel do you think this is a good way to do it, or do I need to make a new function to get lock status?

### CODE REVIEW

**Comment by lfkeitel** [comment]
For later, you can reference a github issue by adding something like `Fixes #3` or `Closes #3` in your commit message.

**Comment by nbebout** [comment]
See how that looks.

File: classes/NamecheapClient.php [hunk 1]
```diff
@@ -57,18 +57,27 @@ public function GetAllDomains() : array {
...
     }
 
+    // will return string true if domain is locked, false if domain is unlocked
+    private function DomainLocked(string $domain) : bool {
+            $queryData2 = $this->baseApiArgs('namecheap.domains.getRegistrarLock');
+            $queryData2['DomainName'] = urlencode($domain);
+            $qs2 = http_build_query($queryData2);
+            $url2 = "{$this->server}{$this->apiEndpoint}?$qs2";
+            $xml2 = simplexml_load_file($url2);
+            return (string)$xml2->CommandResponse->DomainGetRegistrarLockResult->attributes()->RegistrarLockStatus === 'true'? true: false;
```
Comment (lfkeitel) [inline classes/NamecheapClient.php]: This doesn't need to be a ternary. Just use `return (string)($xml2->CommandResponse->DomainGetRegistrarLockResult->attributes()->RegistrarLockStatus) === 'true';`

**Comment by nbebout** [comment]
Ok, I changed that.

**Comment by lfkeitel** [comment]
Like you need my approval...

**Pull request merged**